tests/common.bash

```diff
@@ -22,103 +69,65 @@ function kubectl_apply_should_fail_with_message {
 	fi
 }
 
-function kubectl_apply_should_succeed {
-	run kubectl --context $CLUSTER_CONTEXT apply --wait --timeout $TIMEOUT  -f $1
-	[ "$status" -eq 0 ]
-}
-
-function kubectl_namespace_apply_should_succeed {
-	run kubectl --context $CLUSTER_CONTEXT --namespace $2 apply --wait --timeout $TIMEOUT  -f $1
-	[ "$status" -eq 0 ]
+function kubectl_apply {
```

Comment:
why not keep `kubectl_apply_should_succeed`?

Reply:
to improve readability, I hated function names :)
```bash
kubectl_namespace_apply_should_succeed
kubectl_apply_should_succeed
wait_for_all_cluster_admission_policies_to_be_active
wait_for_all_cluster_admission_policy_condition PolicyActive
wait_for_all_admission_policy_condition
```